packages/core/src/components/select/select.scss

```diff
@@ -53,26 +75,60 @@
         color: helpers.color('content-disabled');
       }
     }
+  }
 
-    &.-borderless {
-      @include mixins.with-inner-focus('action');
-      @include with-padding(); // no border width because border will be unset
+  .ods-select-custom.-open + .ods-icon {
+    transform: rotate(180deg);
+  }
 
-      background-color: transparent;
-      border: 0;
+  .ods-select-custom-dropdown {
+    @include mixins.with-inner-focus('action');
 
-      &:hover {
-        background-color: helpers.color('background-action-subtle-hover');
-      }
+    background-color: helpers.color('background-input');
+    border-radius: helpers.border-radius('small');
+    box-sizing: border-box;
+    display: none;
+    margin-top: helpers.space(0.5);
+    padding: 4px; // spread set on `with-inner-focus` mixin
```

Comment:
```suggestion
    padding: helpers.space(0.5); // spread set on `with-inner-focus` mixin
```

Reply:
The spread is a hardcoded value, based on requirements and calculations within the mixin(s).

I cannot use a space here, because changing a base would mess up the component.

I will however add an extra note to better identify that this must be a hardcoded value.